languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8788,6 +8788,22 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="LA_OU_LES_PLUS" name="la ou les plus" default="temp_off">
+          <pattern>
+              <token postag="V.*" postag_regexp="yes"></token>
+              <marker>
+                  <token regexp="yes">la|les</token>
+              </marker>
+              <token regexp="yes">plus|moins</token>
+              <token min="0" max="2" postag="A.*" postag_regexp="yes"></token>
+              <token>.</token>
```

Comment:
`<token postag="SENT_END" />`

Reply:
If your postag is just `SENT_END`, there's no need for `postag_regexp="yes"`. You can include other punctuation at the end of the sentence: `<token postag="SENT_END" regexp="yes">[\.\?\!]</token>`.

Note that the `.` and the `?` need to be "escaped" with a `\` so the system knows you mean an actual period and an actual question mark (and not the regex elements `.` and `?`)